src/yang/ietf-bgp-common.yang

```diff
@@ -322,6 +327,15 @@ submodule ietf-bgp-common {
         "RFC 2385: Protection of BGP Sessions via the TCP MD5
                    Signature Option.";
     }
+    container bfd {
+      if-feature "bt:bfd";
+      uses bfd-types:client-cfg-parms;
+      description
+        "BFD client configuration.";
+      reference
+        "RFC 9127, YANG Data Model for Bidirectional Forwarding
```

Comment:
This reference should also be updated to point to 9127-bis.

Reply:
I'll make this XXXX for now and open a new issue for updating it after we're through the 9127-bis dance.